### ISSUE TITLE
Bug: HTML tag `lang` attribute missing

### ISSUE THREAD

**Issue by quinnmcphail** [React-Indy/website#29]
## Description

The html tag is missing the `lang` attribute, which causes accessibility issues.

## Steps to Reproduce

1. Go to any page of the site
2. View the source code of the page
3. See that the root html tag does not contain a `lang` attribute

## Expected Behavior

When viewing any page on the site, the lang attribute is present on the root html tag.

## Device & Browser Details

- Device: any
- Operating system: any
- Browser: any

## Additional Context

Additional i18n needs to be considered in the future, but the basic i18n configuration can be set up to accommodate basic accessibility concerns.
